src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/FlyoutTests/UnoSamples_Tests.Flyout.cs

```diff
@@ -134,5 +134,61 @@ public void FlyoutTest_Unloaded()
 			_app.WaitForNoElement(outerButton);
 			_app.WaitForNoElement(innerButton);
 		}
+
+		[Test]
+		[AutoRetry]
+		public void FlyoutTest_Simple_FlyoutsCanBeDismissed()
+		{
+			Run("Uno.UI.Samples.Content.UITests.Flyout.Flyout_Simple");
+
+			var majorStepIndex = 0;
+			var testableFlyoutButtons = new string[]
+			{
+				"LeftFlyoutButton",
+				"RightFlyoutButton",
+				"BottomFlyoutButton",
+				"TopFlyoutButton",
+				//"FullFlyoutButton", // unclosable without native back button/gesture
+				"CenteredFullFlyoutButton",
+				//"FullOverlayFlyoutButton", // unclosable without native back button/gesture
+				"WithOffsetFlyoutButton",
+			}.ToDictionary(x => x, x => _app.Marked(x));
+
+			// initial state
+			_app.WaitForElement(testableFlyoutButtons.First().Value);
+			var initialScreenshot = TakeScreenshot($"{majorStepIndex++} Initial State", ignoreInSnapshotCompare: true);
+
+			var dismissArea = GetDismissAreaCenter();
+			foreach (var button in testableFlyoutButtons)
+			{
+				// show flyout
+				button.Value.Tap();
+				var flyoutOpenedScreenshot = TakeScreenshot($"{majorStepIndex++} {button.Key} 0 Opened", ignoreInSnapshotCompare: true);
```

Comment:
Would be useful to add the key there:

```suggestion
				var flyoutOpenedScreenshot = TakeScreenshot($"{majorStepIndex++} {button.Key} {button.Key} 0 Opened", ignoreInSnapshotCompare: true);
```

Reply:
It is already there. You just duplicated it.